src/views/tokenLaunch/tokenLaunch.tsx

```diff
@@ -84,11 +84,11 @@ function TokenLaunch() {
 
   return (
     <div id="token-launch-view">
-      {networkId != arbitrum_mainnet.chainId ? (
+      {false ? (
```

Comment:
Can you check this? I changed this to check few things. But you can check for different condition instead of just false.